### DISCUSSION TITLE
開機時 App 出現無障礙服務提示彈窗 導致 腳本被阻塞

### DISCUSSION THREAD

**Discussion by nathan-recu** [hamibot/hamibot#59] [问题]
這事情發生過好幾次了

只要在開機時跳出需要無障礙服務的提示彈窗，即使腳本實際上是透過Root權限開啟無障礙服務，也會被阻塞在彈窗畫面上。

有什麼辦法讓Hamibot預設不跳出無障礙服務的彈窗提示嗎？

**Comment by batu1579** [comment]
那个不再提示的选项有用吗？
